deploy/01_genesis.ts

```diff
@@ -33,22 +33,25 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
   }
   if (
     holographGenesisContract != null &&
-    holographGenesisContract.address == '0x9f69aefbb4418a1b4642116b5c8c2b30896019a8'
+    holographGenesisContract.address == '0x0C8aF56F7650a6E3685188d212044338c21d3F73'
   ) {
-    let deployedCode: string = await hre.provider.send('eth_getCode', [holographGenesisContract.address, 'latest']);
+    let deployedCode: string = await hre.ethers.provider.send('eth_getCode', [
+      holographGenesisContract.address,
+      'latest',
+    ]);
     if (deployedCode == '0x' || deployedCode == '') {
       // no deployed code found, we will need to deploy
       if (hre.networkName == 'localhost') {
         // deploy HolographGenesis on localhost
         await hre.ethers.provider.sendTransaction(
           [
             '0x',
-            'f909ad8080830aae608080b9095d608060405234801561001057600080fd5b50',
+            'f909a68080830f42408080b90956608060405234801561001057600080fd5b50',
```

Comment:
can we add  some small comments here about how these giant byte strings were generated

Reply:
Added that in next commit